Lib/test/test_struct.py

```diff
@@ -578,6 +578,14 @@ def test__sizeof__(self):
         self.check_sizeof('0s', 1)
         self.check_sizeof('0c', 0)
 
+    def test_boundary_error_message(self):
+        import ctypes
```

Comment:
I'd suggest moving this import module level and use the following pattern:

```py
try:
    import ctypes
except ImportError:
    ctypes = None
```

then add

```py
@unittest.skipUnless(ctypes, 'requires ctypes')
def test_boundary_error_message(self):
    ...
```